### PR TITLE
Code Quality: Force import of functions / classes

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Try implementing this coding standard. [`SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly`](https://github.com/slevomat/coding-standard/blob/master/doc/namespaces.md#slevomatcodingstandardnamespacesreferenceusednamesonly-)

## Testing Instructions

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

## Checklist

- [ ] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [ ] I have tested this code to the best of my abilities
- [ ] I have verified accessibility to the best of my abilities ([docs](https://github.com/googleforcreators/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [ ] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/googleforcreators/web-stories-wp/tree/main/docs#testing))
- [ ] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

See #12729

### CODE REVIEW

**Comment by swissspidy** [comment]
This is the opposite of the `FullyQualifiedGlobalFunctions` rule we currently use, so we'd have to remove that one in favor of this rule.

File: phpcs.xml.dist [hunk 1]
```diff
@@ -215,6 +215,16 @@
...
+        <property name="allowFullyQualifiedGlobalFunctions" value="true" />
+        <property name="allowFullyQualifiedGlobalClasses" value="true" />
+        <property name="allowFullyQualifiedGlobalConstants" value="true" />
+        <property name="allowFallbackGlobalFunctions" value="true" />
+        <property name="allowFallbackGlobalConstants" value="true" />
+        <property name="allowFullyQualifiedNameForCollidingClasses" value="true" />
+    </properties>
+  </rule>
   <rule ref="SlevomatCodingStandard.Namespaces.UseFromSameNamespace" />
   <!-- All references to types named Exception or ending with Exception must be referenced via a fully qualified name. -->
```
Comment (swissspidy) [inline phpcs.xml.dist]:
```suggestion
```

This comment refers to something inexistent.

File: phpcs.xml.dist [hunk 2]
```diff
@@ -215,6 +215,16 @@
     </properties>
   </rule>
   <!-- Prohibits uses from the same namespace. -->
```
Comment (swissspidy) [inline phpcs.xml.dist]: This comment is for the `UseFromSameNamespace` rule below and needs to be moved down accordingly.

**Review by swissspidy — APPROVED** [review]
Apart from the two nits this LGTM

**Comment by spacedmonkey** [comment]
> SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly

Update comments.

**Review by swissspidy — APPROVED** [review]

**Pull request merged**